src/studio-header/Header.jsx

```diff
@@ -27,29 +27,39 @@ const Header = ({
 }) => {
   const { authenticatedUser, config } = useContext(AppContext);
 
+  const getPagePath = (isMfePageEnabled, urlParameter) => {
+    if (isMfePageEnabled === 'true') {
+      if (urlParameter === 'tabs') {
+        return `/course/${courseId}/pages-and-resources`;
+      }
+      return `/course/${courseId}/${urlParameter}`;
+    }
+    return `${config.STUDIO_BASE_URL}/${urlParameter}/${courseId}`;
```

Comment:
Initially I spot checked this method by going into studio and looking at urls, and then I had this question:

> wouldn't some of these be ${config.STUDIO_BASE_URL}/**settings**/${urlParameter}/${courseId}; like `https://studio.stage.edx.org/settings/grading/course-v1:edX+cheating101+2018T3`?

I then realized that `urlparameter` contains settings/<page type> for those pages which were denoted "settings."

It would create needless complication to remove the "settings" from those new urls, so this makes sense to me.